front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarPhases.tsx

```diff
@@ -33,27 +35,40 @@ const FilterSidebarPhases = ({
     return selectedPhase === id;
   };
 
+  const name = 'phase';
+
   return (
-    <Menu
-      secondary={true}
-      vertical={true}
-      fluid={true}
-      className="intercom-admin-input-manager-phases"
-    >
-      <Menu.Item onClick={clearFilter} active={!selectedPhase}>
```

Comment:
The `active` prop here is replaced by `isSelected` on `FilterRadioButton` (provides the same feature: highlighting on hovering with a dropable element).